pkg/ssh/ssh_keys.go

```diff
@@ -12,17 +12,23 @@ import (
 	"github.com/cli/safeexec"
 )
 
-type SshContext struct {
+type Context struct {
 	ConfigDir string
 	KeygenExe string
 }
 
-type SshKeyPair struct {
+type KeyPair struct {
 	PublicKeyPath  string
 	PrivateKeyPath string
 }
 
-func (c *SshContext) LocalPublicKeys() ([]string, error) {
+type KeyAlreadyExistsError struct {
+	keyPath string
+}
+
+func (err *KeyAlreadyExistsError) Error() string { return "SSH key already exists: " + err.keyPath }
```

Comment:
We're going to ignore this error anyway, so I don't think it's worth it to create a new error type and craft a dynamic error message. Instead, for simplicity, I would just recommending making a new error _instance_ and returning that:

```go
var KeyAlreadyExistsError = errors.New("SSH key already exists")

// in the implementation:
return &keyPair, KeyAlreadyExistsError

// when checking the error:
if err != nil && !errors.Is(err, KeyAlreadyExistsError) {
  return err
}
```